server/sonar-web/src/main/js/apps/issues/HeaderView.js

```diff
@@ -50,11 +50,12 @@ export default Marionette.ItemView.extend({
   },
   serializeData () {
     const me = !!this.options.app.state.get('query').assigned_to_me;
+    // FIXME user is passed down ugly
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)